src/AppBundle/Service/CasrecService.php

```diff
@@ -69,8 +74,14 @@ private function updateCasrecStatsSingle(CasRec $casrec)
             ->setParameter('c2', strtolower($caseNumber))
             ->getResult();
         if ($results && $results[0] instanceof Client) {
+            $client = $results[0]; /* @var $client Client */
+            // last report is currently ordered
+            $submittedReports = $client->getSubmittedReports();
+            $lastReport = count($submittedReports) > 0  ? $submittedReports->first() : null;
```

Comment:
Optional: If report order changes for any reason (getSubmittedReports()) this will screw up the stats. I know its extra work here but is it worth reordering here to ensure the order is still latest first here?

Reply:
getSubmittedReports orders them already. I was thinking about changing the name of that method. It's misleading as getSubmittedReports should just return the reports, If it returns them ordered, than it should be called differently

Reply:
maybe a new method getLatestSubmittedReports(). The change was optional, im just conscious that if any other dev picks up a story to change the ordering (to last updated for example), then they wont be aware of this dependency